server/graphql/v1/mutations/expenses.js

```diff
@@ -340,17 +341,25 @@ export async function editExpense(remoteUser, expenseData) {
       PayoutMethodId !== expense.PayoutMethodId,
     );
 
-    return expense.update(
-      {
-        ...cleanExpenseData,
-        lastEditedById: remoteUser.id,
-        incurredAt: expenseData.incurredAt || new Date(),
-        status: shouldUpdateStatus ? 'PENDING' : expense.status,
-        PayoutMethodId: PayoutMethodId,
-        legacyPayoutMethod: models.Expense.getLegacyPayoutMethodTypeFromPayoutMethod(payoutMethod),
-      },
-      { transaction: t },
-    );
+    if (cleanExpenseData) {
```

Comment:
Why is this condition needed?

Reply:
Not sure how this ended up here, removing it...